src/database/factories/AddressFactory.php

```diff
@@ -2,12 +2,13 @@
 
 use Faker\Generator as Faker;
 use LaravelEnso\Addresses\app\Models\Address;
+use LaravelEnso\Addresses\app\Models\Country;
 
 $factory->define(Address::class, function (Faker $faker) {
     return [
         'addressable_id' => $faker->randomKey,
         'addressable_type' => $faker->word,
-        'country_id' => $faker->randomDigit,
+        'country_id' => Country::inRandomOrder()->first()->id,
```

Comment:
if this approach is not desirable, we can instead use `$faker->randomDigitNotNull`